fv3_cap.F90

```diff
@@ -351,6 +366,7 @@ subroutine InitializeAdvertise(gcomp, rc)
 ! create fcst grid component
 
     if( quilting ) then
+      wrttasks_per_group_from_parent = wrttasks_per_group_from_parent * num_threads
```

Comment:
Does this work with traditional threads?

Reply:
I will run one test without esmf managed threading to confirm.

Reply:
@junwang-noaa @DusanJovic-NOAA was this test completed?

Reply:
Yes, I ran that test manually